lib/parity/environment.rb

```diff
@@ -57,9 +57,9 @@ def deploy_to_heroku
     end
 
     def restore
-      if production?
+      if production? && additional_restore_arguments["--force"].nil?
```

Comment:
Sounds good.

Maybe we can make it a separate private method with a clear name so the intent is clear?